fixed some pending bugs from GSoC 2017 in benchmarking GUI application

raghavendersahdev opened the pull request:
## Changed BenchmarkingGUI app

* modified-benchmarkingGUI app
* fixed all warnings for the benchmarking GUI project
* fixed bugs like: labels for ORB, VO calib/gt files check, console outputs cleared,made Line color consistent, grayscale keypoints colored, etc.
* modified check boxes for true/false values/parameters 

## PR Description

* This PR fixes some bugs, more bugs need to be fixed, I will open a new PR for those bugs in a few days.
---

I acknowledge to have:
* Read the [`CONTRIBUTING`](https://github.com/MRPT/mrpt/blob/master/.github/CONTRIBUTING.md) page
* Updated [`doc/doxygen-pages/changeLog_doc.h`](https://github.com/MRPT/mrpt/blob/master/doc/doxygen-pages/changeLog_doc.h) to describe these changes (if applicable)
* Updated [`AUTHORS`](https://github.com/MRPT/mrpt/blob/master/AUTHORS) (if applicable)

(Notify: @MRPT/owners )

jlblancoc commented:
Thanks! :+1: 

Please, take a look at [this output](https://travis-ci.org/MRPT/mrpt/jobs/281487657) from the Travis test on clang format... 
Please, run: 

        cd MRPT_ROOT
        bash scripts/clang_format_codebase.sh

and using `git status`, use `git add` only those directories on which you work on to include the required format changes.

raghavendersahdev commented:
Thanks @jlblancoc I applied the clang format to my files, added only the files I changed, and rebased my pull request.
Currently the python check passes. In total 3/7 checks pass [here](https://travis-ci.org/MRPT/mrpt/builds/281729892). but the top 4 checks are errored. Is there a way to fix the errored checks or is it fine as is?
I checked some of other pull requests [here](https://travis-ci.org/MRPT/mrpt/pull_requests).  Many are errored, how many / which checks are normally required for a pull request to be merged successfully?

jlblancoc commented:
Thanks! The other Travis errors are not related to your changes, so can be ignored... not the ideal situation, but that's the way it was before your PR, so, no problem on your side! :+1:
